docs/examples/plot_rainbowgram.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""
+================
+Rainbowgrams
+================
+This notebook demonstrates how to use "Rainbowgrams" to simultaneously 
+visualize amplitude and (unwrapped) phase (differential) as demonstrated in the
+`NSynth paper<https://proceedings.mlr.press/v70/engel17a/engel17a.pdf>`
```

Comment:
This link is not correctly formatted - please try to build the docs on your own to check these things.